tests/bluetooth/bt_crypto/src/test_bt_crypto.c

```diff
@@ -8,8 +8,7 @@
 #include <string.h>
 
 #include <zephyr/ztest.h>
-
-#include <bt_crypto.h>
+#include "bt_crypto.h"
```

Comment:
guess `#include <bt_crypto.h>` would be better here, as most compilers probably will fallback to this behavior as this header is not found in same folder as the source file.

Reply:
discussed offline.

Using `""` here to indicate this is a private header for bt_crypto used in tests, which is a valid point.

Reply:
Also disagree with this one. We forcefully add `subsys/bluetooth/crypto` to the include path here, so I think using `""` indicates that this is a "fake local" include and not a global one.